battle_move_effects_test.go

```diff
@@ -10,38 +10,60 @@ var _ = Describe("Stat-changing moves", func() {
 	agent1 := Agent(new(dumbAgent))
 
 	DescribeTable("Move Stat-change data",
-		func(id MoveId) {
-			// move := GetMove(id)
+		func(id MoveId, stat int) {
+			move := GetMove(id)
+			Expect(move.AffectedStat()).To(Equal(uint8(stat)))
 		},
-		Entry("Howl", MoveHowl),
-		Entry("Harden", MoveHarden),
+		Entry("Howl", MoveHowl, StatAtk),
+		Entry("Double Team", MoveDoubleTeam, StatEvasion),
+		Entry("Harden", MoveHarden, StatDef),
 	)
```

Comment:
I think this check should just be moved to the other table so that you don't have to maintain two separate tables.

Reply:
Valid, but this test is just meant to be a sanity check to make sure we are generating code for the data correctly. If this test fails, we know for sure that something is wrong with the code generation.